README.md

```diff
@@ -8,7 +8,7 @@ This is a Home Assistant custom integration for Wi-Fi connected Dyson devices, a
 [![HACS Action](https://badgen.net/github/checks/libdyson-wg/ha-dyson/main/HACS%20Action)](https://github.com/libdyson-wg/ha-dyson/actions/workflows/hacs.yaml)
 [![Latest Commit](https://badgen.net/github/last-commit/libdyson-wg/ha-dyson/main)](https://github.com/libdyson-wg/ha-dyson/commit/HEAD)
 
-## Migration from shenxn/ha-dyson
+## Migration from [shenxn/ha-dyson](https://github.com/shenxn/ha-dyson)
```

Comment:
So this is a _really_ petty nitpick, but I'm hoping someday Google will pick this repo instead of Shenxn's when googling "home assistant dyson" - and I wonder if adding a link to shenxn will work against that goal.

Reply:
I see your point. Initially I wasn't aware of the history part, so I figured this might help others with the confusion. 

If the migration part is moved on the bottom of the readme, I think it wouldn't be much of an issue.

It's your call 😉

Reply:
Heh, yeah it probably doesn't actually matter to me honest. And just for the record, I don't have a beef with shenxn or anything. It's just Google directs most people at his repo which just happens to be unmaintained.

Let's leave the link - you're right, it could possibly help someone.

With regards to the rearrangement, I think your idea to move the migration steps is great. I'm on my phone so I can't do it now, but if you don't want to bother I can get to that tonight when I merge this PR.

Reply:
No worries, I can do it all in this one 👍

Reply:
Also thanks for your contribution! It's always exciting getting PRs 😄

Reply:
Happy to help! All changes done now :wink:

Reply:
Thanks again, I'll get it merged in just a bit.